commons/src/test/java/org/eclipse/kapua/commons/util/ClassUtilTest.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.commons.util;
+
+import org.eclipse.kapua.KapuaException;
+import org.eclipse.kapua.commons.event.ServiceEntry;
+import org.eclipse.kapua.qa.markers.junit.JUnitTests;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.lang.reflect.Constructor;
+
+@Category(JUnitTests.class)
+public class ClassUtilTest extends Assert {
+
+    String serviceName, address;
+
+    @Test
+    public void constructorTest() throws Exception {
+        Constructor<ClassUtil> classUtilConstructor = ClassUtil.class.getDeclaredConstructor();
+        classUtilConstructor.setAccessible(true);
+        classUtilConstructor.newInstance();
+    }
+
+    @Test(expected = NullPointerException.class)
```

Comment:
This is the only test for ClassUtil contructor with two parameters - can you please add additional tests? Some positive scenarios would be nice.